File: java/src/com/company/context/ConfigBeanItem.java
package com.company.context;

class ConfigBeanItem {
    public final String mId;
    public final Class<?> mInterface;
    public final Object mInstance;

    ConfigBeanItem(String id, Class<?> interfaze, Object instance) {
        mId = id;
        mInterface = interfaze;
        mInstance = instance;
    }
}

File: dotNet/Lib/ConfigBeanItem.cs
using System;

namespace Com.Company.Context {
    public class ConfigBeanItem {
        public ConfigBeanItem(string id, Type interfaze, object instance) {
            Id = id;
            Interface = interfaze;
            Instance = instance;
        }

        public string Id { get; private set; }

        public object Instance { get; private set; }

        public Type Interface { get; private set; }
    }
}